core/src/main/java/org/apache/accumulo/core/fate/zookeeper/ZooUtil.java

```diff
@@ -75,7 +74,7 @@ public LockID(String root, String serializedLID) {
         path = root + "/" + sa[0].substring(0, lastSlash);
       }
       node = sa[0].substring(lastSlash + 1);
-      eid = new BigInteger(sa[1], 16).longValue();
```

Comment:
Does anyone know why this was using BigInteger?  Its a slow way to parse a long, not sure if there is a reason for doing it this way.

Reply:
Looked in git history and found issue [ACCUMULO-965](https://issues.apache.org/jira/browse/ACCUMULO-965) as the reason BigInteger was used.  Switched the code to use `Long.parseUnsignedLong` in 3c61d828df70f232994cc592d6a2d184fa700247 whcih I think will solve the same problem that BigInteger was solving but more efficiently.